src/plugins/auto-closer/index.js

```diff
@@ -0,0 +1,172 @@
+
+    return [
+        "is:open",
+        `repo:${owner}/${repo}`,
+        "no:assignee no:milestone no:project -label:accepted",
+        `updated:<${activityCutoffDate.format("YYYY-MM-DD")}`
+    ].join(" ");
+}
+
+/**
+ * Creates the bot comment to leave on auto-closed accepted issues.
+ * @returns {string} comment message
+ * @private
+ */
+function createAcceptedAutoCloseMessage() {
+    return `
+Unfortunately, it looks like there wasn't enough interest from the team
+or community to implement this change. While we wish we'd be able to
+accommodate everyone's requests, we do need to prioritize. We've found
+that accepted issues failing to be implemented after 90 days tend to
+never be implemented, and as such, we [close those issues](https://eslint.org/docs/maintainer-guide/issues#when-to-close-an-issue).
+This doesn't mean the idea isn't interesting or useful, just that it's
+not something the team can commit to.
+
+Thanks for contributing to ESLint and we appreciate your understanding.
+
+[//]: # (auto-close)
+`;
+}
+
+/**
+ * Creates the bot comment to leave on auto-closed unaccepted issues.
+ * @returns {string} comment message
+ * @private
+ */
+function createUnacceptedAutoCloseMessage() {
+    return `
+Unfortunately, it looks like there wasn't enough interest from the team
+or community to implement this change. While we wish we'd be able to
+accommodate everyone's requests, we do need to prioritize. We've found
+that issues failing to reach accepted status after 21 days tend to
+never be accepted, and as such, we [close those issues](https://eslint.org/docs/maintainer-guide/issues#when-to-close-an-issue).
+This doesn't mean the idea isn't interesting or useful, just that it's
+not something the team can commit to.
+
+Thanks for contributing to ESLint and we appreciate your understanding.
+
+[//]: # (auto-close)
+`;
+}
+
+/**
+ * Closes an issue and adds the auto-close label.
+ * @param {probot.Context} context Probot context for the current repository
+ * @param {number} issueNum The issue number on the current repository
+ * @param {string} commentText The text of the comment to post on the issue.
+ * @returns {Promise<void>} A Promise that fulfills when the issue has been archived
+ */
+async function closeIssue(context, issueNum, commentText) {
+    await Promise.all([
+        context.github.issues.edit(context.repo({ number: issueNum, state: "closed" })),
+        context.github.issues.addLabels(context.repo({ number: issueNum, labels: [AUTO_CLOSE_LABEL] })),
+        context.github.issues.createComment(context.repo({ number: issueNum, body: commentText }))
+    ]);
+}
+
+/**
+ * Gets all issues on the current repository that match a search query
+ * @param {probot.Context} context Probot context for the current repository
+ * @param {string} searchQuery A search query to send to the GitHub API
+ * @returns {Promise<issue>} A list of issues that match the query
+ */
+async function queryIssues(context, searchQuery) {
```

Comment:
nit: this function doesnt need to be async.

Reply:
Good point, I'll change that.